IndicoToolkit/AutoReview/AutoReviewFunctions.cs

```diff
@@ -0,0 +1,225 @@
+using System.Collections.Generic;
+
+using IndicoToolkit.Types;
+
+namespace IndicoToolkit.AutoReview
+{
+    /// <summary>
+    /// Class that hosts suite of auto review functions
+    /// </summary>
+    public class AutoReviewFunctions
+    {
+        public AutoReviewFunctions() { }
+
+        /// <summary>
+        /// Determine whether prediction label matches a label in labels or labels list is empty
```

Comment:
could just be me, but I don't really get what this one is doing from the description

Reply:
How about "Determine if the given list of labels is empty or contains the given prediction's label."

Reply:
Also it's a helper function so we could also remove the description and/or make it private